py/jupyterlite/src/jupyterlite/manager.py

```diff
@@ -68,6 +68,9 @@ def _default_addons(self):
             self.log.debug(f"[lite] [addon] [{name}] load ...")
             try:
                 addon_inst = addon.load()(manager=self)
+                if self._is_sys_prefix_ignored(name):
+                    self.log.debug(f"[lite] [addon] ignore sys prefix for [{name}] ...")
+                    addon_inst.ignore_sys_prefix = True
```

Comment:
hm... it's not really part of the API... if provided to `kwargs`, though, traitlets would just drop unknown kwargs on the floor (for good or ill):

```py
addon_kwargs=dict(manager=self)
if self._is_sys_prefix_ignored(name):
    self.log.debug(f"[lite] [addon] ignore sys prefix for [{name}] ...")
    addon_kwargs.update(ignore_sys_prefix=True)
addon_inst = addon.load()(**addon_kwargs)
```